calcom/cal.com
# feat: Pending members selection

## What does this PR do?

Supports assigning pending members to organizations' sub-team events. Based on #9780, thanks for your work @nikospapcom.

Fixes #9745 

## Type of change


## How should this be tested?

1. Create an organization
2. Create a few teams
3. Invite people to those teams
4. Create an event type on the teams you have pending invited members
5. See that you can select pending members (flagged as "pending")

## Mandatory Tasks
